src/containers/Installer.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+
+import InstallerView from '../components/InstallerView'
+import { installApp } from '../actions'
+
+const mapStateToProps = (state, ownProps) => ({
+  slug: state.slug,
```

Comment:
As `slug` and `url` are never set in a reducer, we can avoid relying on global state here. Those are more two properties of InstallerView's internal state. I know it's a little bit different than the way `ProfileView` works but I think that @goldoraf will confirm.